site-manage/site-group-manager/src/main/resources/Messages.properties

```diff
@@ -99,14 +99,14 @@ autogroups.step1.error.noroles=The site has no roles. Please create at least one
 autogroups.step1.info=Let's start with the first step. Please select which site role(s) to include in the groups you are creating.
 autogroups.step1.table.selectallnone=Select All/None
 autogroups.step1.table.role=Role
-autogroups.step2.choose.section=Yes - (Select members from <strong>specific section/rosters</strong>, who fit the role(s) selected in Step 1)
-autogroups.step2.info=You now have the option to draw group members from specific section(s)/Roster(s). Would you like to do this?
-autogroups.step2.info2=Which section(s)/roster(s) should be included in the groups?
-autogroups.step2.no.section=No - (<strong>Include all site participants</strong>, regardless of section/roster, who fit the role(s) selected in Step 1)
-autogroups.step2.error.emptysections=Please select at least one section if you want to use sections.
+autogroups.step2.choose.section=Yes, allow me to select members in the selected roles from specific rosters.
+autogroups.step2.info=Would you like to draw group members from specific rosters?
+autogroups.step2.info2=Which roster(s) should be included in the groups?
+autogroups.step2.no.section=No, include all site participants in the selected roles, regardless of roster.
```

Comment:
Let's keep the strong tags around "include all site participants".

Reply:
Done